src/Compilers/Core/Portable/CodeAnalysis.csproj

```diff
@@ -864,6 +864,7 @@
     <InternalsVisibleTo Include="Microsoft.Build.Tasks.CodeAnalysis" />
     <InternalsVisibleTo Include="VBCSCompiler" />
     <InternalsVisibleTo Include="VBCSCompilerPortable" />
+    <InternalsVisibleToProjectSystem Include="Microsoft.VisualStudio.ProjectSystem.FSharp" />
```

Comment:
Same.  I forgot this was an IVT in the compiler layer.  We don't have IVT in the compiler layer to really any other part of Roslyn for good reasons.